includes/templates/template_default/templates/tpl_index_default.php

```diff
@@ -29,8 +29,20 @@
 <div id="indexDefaultMainContent" class="content"><?php require($define_page); ?></div>
 <?php } ?>
 
-<?php foreach ($tplVars['listingBoxes'] as $tplVars['listingBox']) { ?>
-<?php require($template->get_template_dir($tplVars['listingBox']['formatter']['template'], DIR_WS_TEMPLATE, $current_page_base, 'listingboxes') . '/' . $tplVars['listingBox']['formatter']['template']); ?>
-<?php } ?>
+<?php 
+   function mainBoxSort($a, $b) {
+     if ($a['formatter']['sortMainPage'] == $b['formatter']['sortMainPage'])
+        return 0;
+     if ($a['formatter']['sortMainPage'] < $b['formatter']['sortMainPage'])
+        return -1;
+     return 1;
+   }

```

Comment:
Note: the preference is to *not* put functions in a template file.

Will probably move this out into a static method of the listingBox class in a future refactoring.